README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
+* `Store` page also, allows customers to Purchase/Enable/Manage and learn more about YITH extended plugins like: 
+  1. YITH Booking and Appointment for WooCommerce
+  2. YITH WooCommerce AJAX Search
+  3. YITH WooCommerce Gift Cards
+  4. YITH WooCommerce Wishlist
+  5. YITH WooCommerce Customize My Account Page 
+  6. YITH WooCommerce Ajax Product Filter.
+* Under Store we have a `Products & Services` submenu option which provides following options: 
+  1. Easy access to Add new product/Import CSV of products into your store. 
+  2. Option to setup bookings & appointments or offer gift cards on purchases via YITH extended plugins.
+* Next submenu under Store is `Sales & Promotions`where customer can setup different promotional offers on products in their store. 
+  1. Here we display 13 different campaign setup options offered by `WonderCart` plugin under `All Campaigns` tab. 
+  2. There is also a `Settings` tab where users can customize look & feel of how sales campaigns are displayed on their Store page.
+* Next submenu under Store is `Payments` where we store owner can setup payments option accepted on purchases at Store. 
+  1. There 3 online payment options via payments gateways like PayPal, Stripe & RazorPay(only for India location). 
+  2. We also offer offline payments option via cheque, bank transfer or cash on delivery.
+* Final submenu under Store is `Store details` with following options:
+  1. Store location and complete address, email address of store owner and currency in which products/services are displayed on the store.
+  2. Under `shipping option` tab on the store details page you can connect to Shipping partner like, Shippo (based on your store location).
+  3. Under `tax settings` tab, store owner can choose to enable/disable sales tax. 
+
+
+## Critical Paths
+
+1. Ecommerce Module should display, `Home` page with site preview, links to Hosting brand a/c and links to pending & completed tasks to setup Ecommerce store.
+2. If currently WordPress site is in coming soon mode, WordPress administrator should see visit site/ store CTA and also a button to Launch or go live under `Ready to go live` section. 
+3. `Store` page should give a brief overview of recent customer activity on store. It also has a section to setup `Ecommerce features` with purchase/enable/manage YITH extended plugins. 
+4. YITH PayPal Payment, WonderCart, YITH Stripe Payments and Ecomdash plugins require WooCommerce to be installed and activated. If WooCommerce isn't setup, `Store` tab should ask customers to first Install & activate WooCommerce. `Home` should display warnings for the same.
+5. `Products & Services` submenu must facilitat CTA's to Add new product/Import CSV of products into store. Also disply services offered by YITH extended plugins.
```

Comment:
I wouldn't consider the CTAs a critical path because it is just a link to the functionality. If the link isn't correct, it won't prevent users from performing those actions in their normal location within WooCommerce. I think we can consider the ability to install the YITH plugins might be considered a critical path since that would be the only way for customers to install those if they aren't present.

Reply:
Content updated as stated above. Please review once.